returnn/datasets/concat_files.py

```diff
@@ -13,6 +13,7 @@
 from returnn.util.basic import try_run
 from returnn.util.multi_proc_non_daemonic_spawn import NonDaemonicSpawnContext
 from returnn.config import SubProcCopyGlobalConfigPreInitFunc
+import tree
```

Comment:
I would not put the import as a global import, as I'm not sure all people have installed this. Actually I guess many people have not installed it. It was only recently introduced to RETURNN. So it would break quite a lot of setups. I think it should only be imported when people make use of this dataset.

Reply:
Oh, hm, I see.